app/src/main/res/values/strings.xml

```diff
@@ -333,9 +333,14 @@
 
   <!--Scan Simple ID-->
   <string name="scansimpleid_title">Scan card ID</string>
+
+  <!--- Sync Indicator-->
   <string name="sync_indicator_status_pending">Sync pending</string>
   <string name="sync_indicator_status_failed">Sync failed</string>
   <string name="sync_indicator_status_syncing">Syncing…</string>
   <string name="sync_indicator_status_synced_just_now">Synced just now</string>
   <string name="sync_indicator_status_synced">Synced %d min ago</string>
+  <string name="sync_indicator_dialog_error_network">Make sure you have a working internet connection to sync patient records.</string>
+  <string name="sync_indocator_dialog_button_text">okay</string>
+  <string name="sync_indicator_dialog_error_server">Sync servers are down at the moment. Please try again later or get in touch with your CVHO.</string>
```

Comment:
We generally skip spaces for prefixing resource IDs. E.g.: `bloodpressureentry`, `patientsummary`, etc. Can we fix these strings as well?

Reply:
Sure.